docs/proposals/DataTable/README.md

```diff
@@ -0,0 +1,165 @@
+# DataTable
+
+The `DataTable` Component would give the users options for pagination, sorting,
+and column visibility by leveraging some sort of API or service.
+
+## Design Patterns
+
+Tables are used to organize tabular information to help users understand and
+quickly scan for information.
+
+Some scenarios for Tables include the following:
+
+- Index Pages
+- Dashboards
+
+### Sorting
+
+Sorting can be enabled on column headers to improve usability.
+
+The default sorted column should be based on contextual based on user needs.
+
+To indicate the column being sorted, use an up (ascending) or down (descending)
```

Comment:
are you wording this as an instruction for the engineer to manually control this arrow on their end? Or are you specifying that the component will control the visual of this arrow?

Reply:
I guess we should implement it on the Atlantis side and show the arrows if sorting is enabled on that specific column. I should update the wording here.

Reply:
awesome ^